Sources/ArgumentParser/Usage/MessageInfo.swift

```diff
@@ -73,6 +73,17 @@ enum MessageInfo {
       )
       return
 
+    case let e as NSError:
+      // Send NSError back through the CommandError path
+      self.init(
+        error: CommandError(
+          commandStack: [type.asCommand],
+          parserError: .userValidationError(e)
+        ),
+        type: type
+      )
+      return
+
```

Comment:
It turns out this bit isn't necessary! The default case handles general errors just fine.